src/media/css/elements--categories.styl

```diff
@@ -0,0 +1,115 @@
+@import 'lib';
+
+// Originally from elements--nav.styl.
+$color--nav--child-bg = #1A95FC;
+
+mkt-category-list {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    list-style: none;
+    width: 100%;
+
+    +media--wide-tablet() {
+        flex-direction: row;
+        justify-content: center;
+    }
+}
+
+mkt-category-item {
+    width: 100%;
+
+    +media--wide-tablet() {
+        margin: 8px;
+        width: auto;
+    }
+}
+
+.mkt-category-link {
```

Comment:
I named the class what I would name the element if I created it ```<mkt-category-link>```. Although we have tons of listeners for just ```<a>``` (such as for navigate) so I left it as an ```<a>```